src/vs/workbench/parts/files/browser/views/explorerViewer.ts

```diff
@@ -111,6 +161,55 @@ export class FileDataSource implements IDataSource {
 		}
 	}
 
+	private resolveVirtualDirectories(files: FileStat[]) {
+		if (!this.enableVirtualDirectories) {
```

Comment:
Based on the previous comment, this wouldn't be needed since it's already checked; however, we'd need a way to check to see if we've already done the calculations on this folder or not